### PR TITLE
fix(component/forms): Enumeration widget : fix blur and header when edit

### PR DESCRIPTION
**Pull request by Sylvain-Talend**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
- Possible to edit multiple values at the same time
- Header "Edit a value" when editing an item no more displayed

**What is the new behavior?**
- Not Possible to edit multiple values at the same time
- Header "Edit a value" when editing an item displayed

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

**Comment by Sylvain-Talend** [comment]
Ok it is fine for me, I reverted this change

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

File: packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js [hunk 1]
```diff
@@ -303,6 +303,11 @@ class EnumerationWidget extends React.Component {
 		}
 	}
 
+	getItemsInDefaultMode() {
```
Comment (jlame) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: what about calling it resetItems ?

Comment (jlame) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: add a comment explaining why we need this

Comment (jlame) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: add test for this function

Comment (Sylvain-Talend) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: done

Comment (Sylvain-Talend) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: done

Comment (Sylvain-Talend) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: done and moved to utils.js to not make EnumerationWidget too heavy

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://313.talend.surge.sh)

**Pull request merged**